app/src/main/res/values/strings.xml

```diff
@@ -16,10 +16,16 @@
     <string name="close">Uždaryti</string>
 
     <string name="problem_search">Problemos paieška</string>
+    <string name="progress_dialog_loading">Kraunama</string>
 
     <string name="profile_hint_email">E-mail</string>
     <string name="profile_hint_name">Vardas</string>
     <string name="profile_telephone">Telefonas</string>
+    <string name="message_notsigned">Norint pasiekti profilio informacija prašome prisijungti</string>
+    <string name="title_notsigned">Vartotojas neprisijungęs</string>
+    <string name="problem_send">Pranešti apie problemą</string>
+    <string name="login_sign_in">Sign in with Google</string>
```

Comment:
@ViliusKraujutis for the next release - yes. The further releases need to have English as well for foreigners living in Vilnius